ckanext/canada/templates/public/datatables/datatables_view.html

```diff
@@ -1,6 +1,20 @@
 {% ckan_extends %}
 
+{% block bodytag %}
+  {{- super() -}}
+  {% if resource_view.get('canada_disabled_view') %}
+    id="canada-disabled-view-body"
```

Comment:
should this be a class? seems odd to use an id here

Reply:
I believe I had to do this because of some plugin priority thing in which the class attribute gets overridden. I can just change this to a `data` attribute though, like `data-view-disabled`

Reply:
data attribute works